typing/oprint.ml

```diff
@@ -100,8 +100,21 @@ let print_out_value ppf tree =
     | Oval_nativeint i -> fprintf ppf "%nin" i
     | Oval_float f -> pp_print_string ppf (float_repres f)
     | Oval_char c -> fprintf ppf "%C" c
-    | Oval_string s ->
-        begin try fprintf ppf "%S" s with
+    | Oval_string (s, maxlen, kind) ->
+       begin try
+               let len = String.length s in
+                fprintf ppf "%s%S%s"
```

Comment:
```
# type t = A of bytes;;
type t = A of bytes
# A "foo";;
- : t = A Bytes.of_string "foo"
# A Bytes.of_string "foo";;
Error: Syntax error
```
I'd like the output to be pasteable into the toplevel, so it should be syntactically correct.